server/trillian_log_server/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM golang
+
+ENV DB_USER test
+ENV DB_PASSWORD zaphod
+ENV DB_DATABASE test
+ENV DB_HOST 127.0.0.0:3306
+
+ENV HOST 0.0.0.0
+ENV RPC_PORT 8090
+ENV HTTP_PORT 8091
+
+ENV DUMP_METRICS 0s
+
+ADD . /go/src/github.com/google/trillian
+WORKDIR /go/src/github.com/google/trillian 
+
+RUN go get -d -v ./server/trillian_log_server
+RUN go install ./server/trillian_log_server
```

Comment:
Why do you tell `go get` not to install `trillian_log_server` (using the -d flag), then immediately install it using `go install`?